src/Analysis/Ast/Impl/Resources.resx

```diff
@@ -189,4 +189,7 @@
   <data name="UnableToDetermineCachePathException" xml:space="preserve">
     <value>Unable to determine analysis cache path. Exception: {0}. Using default '{1}'.</value>
   </data>
+  <data name="TypeVarNoArguments" xml:space="preserve">
+    <value>TypeVar must have at least one argument.</value>
```

Comment:
Should be a bit more specific. What if user types `T = TypeVar(int, float, complex)` missing `T` per https://www.python.org/dev/peps/pep-0484? The message will then be confusing since there are 3 arguments there. It should be something like `TypeVar requires first argument to be a string with the same name as the variable.`. Jake may suggest something too.

Reply:
This is actually handled by another case, this diagnostic message is specifically for when TypeVar is called with no arguments.

Reply:
May be too granular. Users probably don't need that much granularity.

Reply:
I think having a no-arg case is alright.